diagnostic.rb

```diff
@@ -1,9 +1,65 @@
 # Question 1. Paste your code below.
+class Pets < ActiveRecord::Migration
+  def change
+    create_table :pets do |t|
+      t.date :born_on
+      t.string :kind
+      t.string :name
+
+      t.timestamps null: false
+    end
+  end
+end
+
+class AddPersonToPets < ActiveRecord::Migration
+  def change
+    add_reference :pets, :person, references: :people, index: true
+    add_foreign_key :pets, :people, column: :people_id
```

Comment:
all you need to do is 
```
add_reference :pets, :person, index: true, foreign_key: true
```
This will mess up because your saying the foreign key is on the column people_id but it will actually be in the column person_id